packages/lab/src/typography/Text.css

```diff
@@ -103,35 +103,42 @@ h4.uitkText small,
 /* Label-Caption */
 label.uitkText,
 .uitkText-label {
+  --uitkText-line-height: var(--uitk-text-caption-lineHeight);
   font-size: var(--uitk-text-caption-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-caption-lineHeight));
 }
 
 /* Help text */
 .uitkText-helpText.uitkText {
-  font-style: var(--uitk-text-help-fontStyle);
+  --uitkText-line-height: var(--uitk-text-help-lineHeight);
   color: var(--uitk-text-secondary-foreground);
+  font-style: var(--uitk-text-help-fontStyle);
   font-size: var(--uitk-text-help-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-help-lineHeight));
 }
 
 /* Figure 1 */
 .uitkText-figure1.uitkText {
+  --uitkText-line-height: var(--uitk-text-figure1-lineHeight);
   font-size: var(--uitk-text-figure1-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure1-lineHeight));
   font-weight: var(--uitk-text-figure1-fontWeight);
 }
 
 /* Figure 2 */
 .uitkText-figure2.uitkText {
+  --uitkText-line-height: var(--uitk-text-figure2-lineHeight);
   font-size: var(--uitk-text-figure2-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure2-lineHeight));
   font-weight: var(--uitk-text-figure2-fontWeight);
 }
 
 /* Figure 3 */
 .uitkText-figure3.uitkText {
+  --uitkText-line-height: var(--uitk-text-figure3-lineHeight);
   font-size: var(--uitk-text-figure3-fontSize);
-  line-height: var(--uitk-text-lineHeight-backwardsCompat, var(--uitk-text-figure3-lineHeight));
   font-weight: var(--uitk-text-figure3-fontWeight);
 }
+
+.uitkText {
+  line-height: var(--uitkText-line-height);
```

Comment:
to follow our conventions, we should have line-height: var(--uitkText-lineHeight, var(--text-lineHeight)) and set the local variable (--text-lineHeight) throughout this file, so consumers can override with the given API (---uitkText-lineHeight)

Reply:
we should similarly do this for font weight and font size if we are going to do it for line height

Reply:
is this something the consumers should override?

Reply:
internally probably not, but i would say yes for general purpose..
@heswell what are your thoughts here?

Reply:
see comment above